src/theme/dojo/list-item.m.css

```diff
@@ -9,12 +9,13 @@
 	white-space: nowrap;
 	align-items: center;
 	justify-content: flex-start;
-	height: calc(var(--grid-base) * 5.5);
+	min-height: calc(var(--grid-base) * 5.5);
 	display: flex;
 }
 
 .active {
 	border: 1px solid var(--color-highlight-border);
+	transition: none !important;
 }
 
 .selected {
```

Comment:
I think that the dojo theme should have a `movedUp` / `movedDown` class as well as the sizes may change / differ from the default styling.